purple-pill/js/factory.js

```diff
@@ -1,7 +1,7 @@
 app.factory("sharedScope", function($rootScope) {
     var scope = $rootScope.$new(true);
     scope.data = {
-        projectName : "SEAN",
```

Comment:
@simonegli is changing the name to `Canola` due to another typeface already published with the name "Sean"